### ISSUE TITLE
(Opinion) The current site is toxic for the rust community

### ISSUE THREAD

**Issue by fritzblue** [rust-lang/arewewebyet#237]
It is my most humble—and hopefully constructive—opinion that the site (in its current form as of posting) is toxic for the rust community, especially for newcomers considering using rust for web work.

Let me explain.

Go to Google right now and google "rust web" or something similar. I guarantee AWWY will be one of the first results.

Now re-read some of the copy on this SEO-dominating page. Anyone considering using rust for web work must grapple with pessimistic phrases like "_still_ scarce", "_still_ unmet and immature", and my personal pessimistic pet peeve "_Looking further, it doesn't necessarily get better_".

I would argue that this site should strike a tone of (1) evangelizing rust's [already world class web achievements](https://www.techempower.com/benchmarks/#section=data-r18&hw=ph&test=plaintext), (2) opening people's eyes to the wonders of rust for aspects of the web they may have not even considered yet, [such as WebAssembly](https://rustwasm.github.io/book/), and (3) galvanizing potential contributors to action in the few areas where the language/platform does currently fall short (which are much fewer and farther between than the current site would suggest).

Put simply, the site as it stands today acts primarily as a deterrent to any who would seriously consider rust as a language for web development, when in reality anyone choosing rust would be lucky to have stumbled upon such a performant, stable platform that will save them both time (great web libraries and powerful compile-time checks, among other things) and money (fewer server resources due to world-leading performance).

Even as a rust enthusiast myself, it took me many months to venture into web work _almost entirely due to AWWY_. Now I find myself wishing I had ventured sooner.

Please consider dramatically revamping the copy to strike a more _excited_ tone about what I would argue is the most _exciting_ language for web development to come along in years.

Thank you.

**Comment by mindplay-dk** [comment]
Yeah, I'm considering Rust, and almost didn't, because of this site. I only became sceptical of the site when I noticed the news stream basically ended in 2016.

After finding this post, I am now encouraged to keep looking, but no thanks to (or rather in spite of) this site. It does seem rather toxic and probably drives a lot of people away before they ever really look. 😕

**Comment by frederikhors** [comment]
I agree too! Please change that page or put it offline temporarily!

**Comment by ibraheemdev** [comment]
@fritzblue @frederikhors @mindplay-dk The home page has _finally_ been updated. Do you think this can be closed, or are there any other issues?

**Comment by fritzblue** [comment]
Love the updates, happy to close as resolved :heart:
